app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivityCurrentPlace.kt

```diff
@@ -18,6 +18,7 @@ class MapsActivityCurrentPlace(var NameOfCity: TextView? = null) {
     val MIN_TIME: Long = 5000
     val MIN_DISTANCE = 1000f
     val REQUEST_CODE = 101
+    private val PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION: Int = 0
```

Comment:
Moved constant declaration to be accessible in multiple functions and should not be modified after initialization as recommended from: https://kotlinlang.org/docs/coding-conventions.html#idiomatic-use-of-language-features

Reply:
Good.